dao/src/main/resources/db/migration/V90__addMissingScanSummaryIndex.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS scan_summaries_issues_scan_summary_id_idx ON scan_summaries_issues (scan_summary_id);
```

Comment:
Fine with me, just one question: Why is 'IF NOT EXISTS` used here, as we do not use such constraints anywhere else?